### ISSUE TITLE
Erlang 22 support

### ISSUE THREAD

**Issue by JJJollyjim** [rusterlium/erlang_nif-sys#37]
Erlang 22 is out (and in my case, my linux distro has updated), and with it comes NIF version 2.15. This means that NIFs won't build even with the latest `erlang_nif-sys`.

**Comment by grahamhay** [comment]
What needs to be done to implement Erlang 22 support?

**Comment by goertzenator** [comment]
Since I am not so involved in the project any more, I've sketched out the update process [here](https://github.com/rusterlium/erlang_nif-sys/wiki/Updating-this-package-for-new-OTP).

**Comment by OvermindDL1** [comment]
Any progress on this?  A server went down when it received its automatic update as it wasn't able to rebuild the NIF's and so had to manually roll it back, which was not at all fun...

**Comment by filmor** [comment]
Support is on rustler `master`. `erlang_nif_sys` was vendored in there as `erl_nif_sys`, in which the changes have been made.

**Comment by OvermindDL1** [comment]
@filmor Oh awesome, rustler is what I was needing, but any chance of PR'ing the changes back here for those that use this directly?  Or by vendored in do you mean you merged the PR that was here?

**Comment by filmor** [comment]
I don't think that would make much sense, seeing that further development will happen on the rustler repo. I'll have a look.

The PRs in question are https://github.com/rusterlium/rustler/pull/212 and https://github.com/rusterlium/rustler/pull/226.

**Comment by filmor** [comment]
See #39.
